src/tapis-app/src/App/App.tsx

```diff
@@ -61,8 +67,73 @@ const App: React.FC = () => {
       setSelectedSystem(system);
     },
     [setSelectedSystem]
-  )
+  );
```

Comment:
Streams should have its own [Section component](https://github.com/tapis-project/tapis-ui/tree/main/src/tapis-app/src/Sections) rather than be included at the top level of the App.